lib/subscription.rb

```diff
@@ -0,0 +1,23 @@
+class Subscription
+  def self.keyname(service)
+    "claim_review_webhooks_#{service}"
+  end
+
+  def self.add_subscription(service, url)
+    REDIS_CLIENT.sadd(self.keyname(service), url)
+  end
+  
+  def self.remove_subscription(service, url)
+    REDIS_CLIENT.srem(self.keyname(service), url)
+  end
+
+  def self.get_subscriptions(service)
+    REDIS_CLIENT.smembers(self.keyname(service)) || []
+  end
+
+  def self.notify_subscribers(service, claim_review)
+    self.get_subscriptions(service).each do |webhook_url|
+      RestClient.post(webhook_url, {claim_review: claim_review}.to_json)
+    end
+  end
```

Comment:
Does it mean that we send one HTTP request for each new claim review that was fetched? If so, I'm fine with that as long as we don't get too many claim reviews in an update

Reply:
Yes that's the case right now. On a very busy day we average something like a couple hundred new claims across all services, so I don't think that it's enough incoming data to warrant bulk responses quite yet - we can definitely do something like that with redis queues though.

Reply:
OK, let's keep an eye if it becomes an issue.